pkg/appfile/storage/driver/driver.go

```diff
@@ -0,0 +1,26 @@
+package driver
+
+import (
+	"github.com/oam-dev/kubevela/pkg/appfile"
+	"github.com/oam-dev/kubevela/pkg/appfile/template"
+)
+
+// Driver is mutli implement interface
+type Driver interface {
+	// List applications
+	List(envName string) ([]*RespApplication, error)
+	// Save application
+	Save(app *RespApplication, envName string) error
+	// Delete application
+	Delete(envName, appName string) error
+	// Get application
+	Get(envName, appName string) (*RespApplication, error)
+	// Name of storage driver
+	Name() string
+}
+
+// RespApplication will get  application from Storage client
+type RespApplication struct {
```

Comment:
this is exactly the same with `Application`, why don't use that?

Reply:
Repeat import

Reply:
Why not move `Application` out?

Reply:
Better to avoid this duplicate type

Reply:
Because I don't know where it should move. Can you give me some advice? 0.0

https://github.com/oam-dev/kubevela/blob/master/pkg/application/app.go#L29

Reply:
You can just put it here if you don't know

Reply:
I try two ways to resolve it：
1. [move application/Application to storage/driver pkg ](https://github.com/oam-dev/kubevela/pull/776/commits/a4b52935221c6fb3088bc44f8baf8d39864b1fec)
2. [convert storage/driver.RespApplication to application/Application](https://github.com/Cweiping/kubevela/commit/de82405cb366456be13d0368512a1ffb5473332f)：
 ```Go
type Application struct {
	driver.RespApplication
}
```



Waiting for your comments.